default-input.js

```diff
@@ -15,6 +15,12 @@ function niceName (n) {
   return n.replace(/^node-|[.-]js$/g, '').toLowerCase()
 }
 
+function conf (name) {
+  var a = name
+  var b = name.split('.').join('-')
+  return config.get(a) || config.get(b)
```

Comment:
if you wrote it as `return config.get(a) || config.get(a.split(".").join("-"));` you would lose a bit of readability but it would only do the split/join if `config.get(a)` failed.